docker-images/kafka/kafka-thirdparty-libs/2.5.x/pom.xml

```diff
@@ -181,5 +182,25 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Open Policy Authorization plugin -->
+        <dependency>
+            <groupId>com.bisnode.kafka.authorization</groupId>
+            <artifactId>opa-authorizer</artifactId>
+            <version>${opa-authorizer.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.module</groupId>
+                    <artifactId>jackson-module-scala_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka_2.12</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.google.code.findbugs</groupId>
+                    <artifactId>jsr305</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
     </dependencies>
 </project>
```

Comment:
Did you mean to check in `docker-images/kafka/opa-authorizer-0.3.0-all.jar`

Reply:
Thanks for noticing. That was for sure not intentional - left over from the original PoC when the library was not in Maven.